sdk/version/version_base.go

```diff
@@ -8,7 +8,7 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
-	Version           = "1.10.10"
-	VersionPrerelease = ""
+	Version           = "1.10.11"
+	VersionPrerelease = "dev1"
```

Comment:
We don't use the prerelease part for minor releases.
```suggestion
	VersionPrerelease = ""
```

Reply:
Ah, thanks - the Confluence page seemed to indicate I should, so thanks for the correction!

Reply:
Reviewing the checklist, it's a bit confusing - I'll update that part in the template as part of a scheduled overhaul I need to work on.